src/EditorFeatures/Core/Implementation/IntelliSense/SignatureHelp/Controller.Session_UpdateModel.cs

```diff
@@ -76,25 +80,25 @@ private static int GetSelectedParameter(SignatureHelpItem bestItem, int paramete
                     var result = filteredItems.FirstOrDefault(i => IsApplicable(i, argumentCount, name, isCaseSensitive));
                     if (result != null)
                     {
-                        return result;
+                        return (result, false);
                     }
 
                     // if we couldn't find a best item, and they provided a name, then try again without
                     // a name.
                     if (name != null)
                     {
-                        return GetBestItem(currentItem, filteredItems, selectedParameter, argumentCount, null, isCaseSensitive);
+                        return GetBestItem(currentItem, userSelected, filteredItems, selectedParameter, argumentCount, null, isCaseSensitive);
```

Comment:
is it right to pass userSelected along here?

Reply:
Fixed. Thanks